### PR TITLE
ref(ts): Convert `<RangeSlider>` to typescript

### PR DESCRIPTION
**Pull request by billyvg**
Just props to typescript

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/components/forms/controls/rangeSlider.tsx [hunk 1]
```diff
@@ -277,7 +329,7 @@ const Slider = styled('input')`
   }
 
   &::-ms-fill-upper {
-    background: ${p => p.theme.borderLight});
+    background: ${p => p.theme.borderLight};
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/components/forms/controls/rangeSlider.tsx]: GJ TS

Comment (dashed) [reply src/sentry/static/sentry/app/views/settings/components/forms/controls/rangeSlider.tsx]: did typescript really detect this? 🤔

**Comment by billyvg** [comment]
I think the new percy snapshot is desired

File: src/sentry/static/sentry/app/views/settings/components/forms/controls/rangeSlider.tsx (hunk 1 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/views/settings/components/forms/controls/rangeSlider.tsx]: ts-styled-plugin did, yeah.

**Review by markstory — APPROVED** [review]

**Pull request merged**